tools/web.py

```diff
@@ -46,6 +46,10 @@ def generate(env):
     env.Append(CCFLAGS=["-sSUPPORT_LONGJMP='wasm'"])
     env.Append(LINKFLAGS=["-sSUPPORT_LONGJMP='wasm'"])
 
+    # Use big int
+    env.Append(CCFLAGS=["-sWASM_BIGINT"])
+    env.Append(LINKFLAGS=["-sWASM_BIGINT"])
```

Comment:
Thanks, I'll update pr to do the same.

Reply:
```suggestion
    env.Append(LINKFLAGS=["-sWASM_BIGINT"])
```

Since we need to build gdextensions with emscripten => 3.1.62 (better if == 3.1.62) to support gdextensions, we don't need to check the compiler version or `proxy_to_pthread` (in fact, I think we should remove those conditionals in upstream Godot to, and always require `emscripten => 3.1.62`).

Reply:
@Faless I'll create a PR for this for the main repo.

Reply:
Looks like we're still waiting on the `CCFLAGS` line to be removed. But once that's done, I think this should be OK to merge!

Reply:
I've opened #1603 because this can't be delayed much longer, since it breaks all web GDExtensions.